content/id/docs/cluster-administration/proxies.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Berbagai Proxy di Kubernetes
```

Comment:
nit: <i>Proxy</i>

Reply:
not possible to do italic on title template